--- a/volumetric-checkout/volumetric.service
+++ b/volumetric-checkout/volumetric.service
@@ -1,9 +1,10 @@
 [Unit]
 Description=Versioning System for Docker Volumes
-After=docker.service
-Requires=docker.service
+After=podman.service
+Requires=podman.service
 
 [Service]
+Environment=DOCKER_HOST=unix://run/podman/podman.sock
 ExecStart=/usr/bin/volumetric-checkout
 Type=oneshot
 RemainAfterExit=yes
